docs/spec.md

```diff
@@ -4977,12 +4977,14 @@ Performs element-wise arithmetic right-shift operation on the `lhs` tensor by
 #### Examples
 
 ```mlir
-// %lhs: [-1, -128, -36, 5, 3, 7]
-// %rhs: [1, 2, 3, 2, 1, 3]
-%result = "stablehlo.shift_right_arithmetic"(%lhs, %rhs): (tensor<6xi8>, tensor<6xi8>) -> tensor<6xi8>
-// %result: [-1, -32, -5, 1, 1, 0]
+// %lhs: [-1, 0, 1]
+// %rhs: [1, 2, 3]
+%result = "stablehlo.shift_right_arithmetic"(%lhs, %rhs): (tensor<3xi64>, tensor<3xi64>) -> tensor<3xi64>
```

Comment:
The input `lhs = 1` and `rhs = 3` add some value for left shift. For right shift, it would be better to have an input like shiftArithRight(16, 3) = 1, to exemplify the function. This would also include changing the ODS and interpreter tests.